src/components/AppBar/NotificationSnackbar.js

```diff
@@ -54,8 +54,9 @@ class NotificationSnackbar extends PureComponent {
                 key="close"
                 color="inherit"
                 onClick={() => {
-                    this.props.closeSnackbar(key);
+                    size = 'large' > key;
```

Comment:
What's going on with this string comparison and assignment? Why does it no longer call `closeSnackbar`?

Reply:
I am not sure what's happening here, but the onClick is no longer needed as default behavior is to close now. I ran the script to upgrade so I guess it did something weird.